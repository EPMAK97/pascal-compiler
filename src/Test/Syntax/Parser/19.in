function x : integer;
Begin
end
Begin
end.