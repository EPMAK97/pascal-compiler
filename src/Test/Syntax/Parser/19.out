Error in pos 4:1 required ; but found Begin
